site/src/components/steps.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Steps
+description: The steps component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com/design/7raji0IWkuBRxs5kLiADgI/Copilot-Web-UI-kit?node-id=70108-9255&node-type=frame&t=G0FoPLEXPee1yysa-0
+classType: Component
+classPrefixes:
+  - steps
+  - step
+---
+
+# Steps
+
+Used to indicate the user's location in a finite progression. These steps are not interactive.
+
+For example, as a user progresses through 5 items, you would see something like this:
+
+```html
+<div class="steps padding-xxs">
+	<div class="step step-selected"></div>
+	<div class="step"></div>
+	<div class="step"></div>
+	<div class="step"></div>
+	<div class="step"></div>
+</div>
+<div class="steps padding-xxs">
+	<div class="step"></div>
+	<div class="step step-selected"></div>
+	<div class="step"></div>
+	<div class="step"></div>
+	<div class="step"></div>
+</div>
+<div class="steps padding-xxs">
+	<div class="step"></div>
+	<div class="step"></div>
+	<div class="step step-selected"></div>
+	<div class="step"></div>
+	<div class="step"></div>
+</div>
+<div class="steps padding-xxs">
+	<div class="step"></div>
+	<div class="step"></div>
+	<div class="step"></div>
+	<div class="step step-selected"></div>
+	<div class="step"></div>
+</div>
+<div class="steps padding-xxs">
+	<div class="step"></div>
+	<div class="step"></div>
+	<div class="step"></div>
+	<div class="step"></div>
+	<div class="step step-selected"></div>
+</div>
+```
```

Comment:
I'm thinking that that our examples need to be paired with some element containing the description of the current state of the steps: such as `<p class="visually-hidden">Step 1 of 5</p>`.

```suggestion
```

## Accessibility concerns

Although the step component is not interactive, it is also not entirely presentational. Thus, it's recommended that you put `aria-hidden` on the component, while including a `aria-hidden` element that communicates the current state of the user's progress through the steps.
```

Reply:
Nit to @wibjorn's suggestion, this should be

```diff
-while including a `aria-hidden` element that communicates the current state of the user's progress through the steps.
+while including a `visually-hidden` element that communicates the current state of the user's progress through the steps.
```

Reply:
like the nit

Reply:
since I was exactly wrong :D